native_client/dotnet/README.rst

```diff
@@ -119,6 +120,13 @@ There's one last command to run before building, you need to run the `configure.
 
 At this point we are ready to start building the ``native_client``, go to ``tensorflow`` sub-directory, following our examples should be ``D:\cloned\DeepSpeech\tensorflow``.  
 
+
+.NET Core
+~~~~~~~~~
+
+To run it on different platforms - build appropriate ``.so`` file for windows/linux/osx etc. using official `docs <https://deepspeech.readthedocs.io/en/latest/BUILDING.html>`_
```

Comment:
I think you should use internal linking here: ``docs <BUILDING>`_`

Reply:
just tried - didn't work

Reply:
set to link again

Reply:
@lissyx I tried a couple of options, they didn't work, and I see cross files referencing not working in [using docs](https://github.com/mozilla/DeepSpeech/blob/master/doc/USING.rst) too, so if you have some example or reference that might be helpful  - that would be great.

Reply:
See the `ref` stuff

Reply:
it's doesn't render correctly and do not pointing anywhere (same as in link `using docs` ion comment above), I tried `ref` `any` etc

Reply:
I tried as mentioned [here](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#internal-and-external-links)

Reply:
Please trust us, if it's used, it works. Github rendering is broken often. Check the docs task in taskcluster group it has the sphinx generated doc and is the only valid source of truth before readthedocs.io rendering

Reply:
ok, I thought github should render correctly. Will leave it with `ref`, thanks